File: src/main/java/com/raejz/sc/exception/dto/ErrorDTO.java
package com.raejz.sc.exception.dto;

import java.util.List;
import lombok.Data;

/**
 *
 */
@Data
public class ErrorDTO {

  private int status;
  private String errorCode;
  private String message;
  private List<String> errors;
  private String timestamp;
}

File: src/main/kotlin/com/raejz/sc/exception/dto/ErrorDTO.kt
package com.raejz.sc.exception.dto

/**
 *
 */
data class ErrorDTO (
        var status: Int = 0,
        var errorCode: String? = null,
        var message: String? = null,
        var errors: List<String>? = null,
        var timestamp: String? = null
)